File: Data Structures & Algorithms/merge-triplets-to-form-target/submission-1.cpp
class Solution {
public:
    bool mergeTriplets(vector<vector<int>>& triplets, vector<int>& target) {
        bool a = false, b = false, c = false;

        for(auto elem: triplets){
            if(elem[0] > target[0] || elem[1] > target[1] || elem[2] > target[2]) continue;

            if(elem[0] == target[0]) a = true;
            if(elem[1] == target[1]) b = true;
            if(elem[2] == target[2]) c = true;
        }
        return a && b && c;
    }
};

File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        i, j = 0, len(numbers) - 1
        while( i < j ):
            sum = numbers[i] + numbers[j] - target
            if sum == 0:
                return [i+1, j+1]
            elif sum > 0:
                j -= 1
            else:
                i += 1
        return [-1, -1]
        